src/main/java/com/google/cloudassets/discovery/TableCreation.java

```diff
@@ -0,0 +1,119 @@
+package com.google.cloudassets.discovery;
+
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloudassets.discovery.exceptions.ConfigTableException;
+import com.google.cloudassets.discovery.exceptions.TableCreationException;
+
+import static com.google.cloudassets.discovery.Main.executeStringQuery;
+
+/**
+ * This class generates a DDL create table statement for a given table name (based on the
+ * Asset_Tables_Config table).
+ */
+public class TableCreation {
+    private static final String GET_TABLES_CONFIG_QUERY = "SELECT columnName, columnType, "
+                                                        + "isNotNull, isPrimaryKey, "
+                                                        + "allowCommitTimestamp "
+                                                        + "FROM Asset_Tables_Config";
+    // The order of the primary keys in the create table statement matters
+    private static final String GET_COMMON_PRIMARY_KEYS_QUERY = "SELECT columnName "
+                                                        + "FROM Asset_Tables_Config "
+                                                        + "WHERE assetTableName = 'forAllAssets' "
+                                                        + "and isPrimaryKey = True "
+                                                        + "ORDER BY primaryKeyIndex";
+
+    private static StringBuilder createStatement;
```

Comment:
Can you make this non-static? Static variables are essentially global (in the sense that all the objects share them) and we usually want to build code that can be used in parallel for different purposes (e.g. even if we don't plan to use it more than once in our system, some tests may want to instantiate it multiple times, or a future refactoring of the code may need this ability)